ansible/roles/dev/files/django.bashrc

```diff
@@ -0,0 +1 @@
+export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
```

Comment:
Maybe set this as an environment variable inside the common/tasks/main.yml
http://docs.ansible.com/ansible/faq.html#how-can-i-set-the-path-or-any-other-environment-variable-for-a-task-or-entire-playbook